web/models.py

```
@@ -3,7 +3,9 @@
 import string
 import time
 import uuid
+import logging
 from io import BytesIO
+from typing import Any, Optional, ClassVar
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)